Dockerfile

```diff
@@ -46,6 +43,11 @@ RUN curl -o install.py https://azurecliprod.blob.core.windows.net/install.py &&
   printf "\n\n\n\n" | python3 ./install.py && \
   rm ./install.py
 
+RUN helm_version=v3.1.1 \
```

Comment:
Nitpicking:
`helm` is not needed on the image neither for cluster creation nor for operation, the same way that `kubectl` or `cf-cli` aren't, since for now they get downloaded on `buildir/bin/` and pinned to the correct versions needed by the backend. So I would move this block under the `#extras` comment.

Reply:
I think I added this because of this 
```Installing CFO from: https://s3.amazonaws.com/cf-operators/release/helm-charts/cf-operator-3.3.0%2B0.gf32b521e.tgz
/tmp/build/b0d51b9f/catapult/include/func.sh: line 251: helm: command not found
make[1]: *** [Makefile:17: install] Error 127
make: *** [Makefile:182: scf] Error 2```
https://concourse.suse.dev/teams/main/pipelines/prabal-demo-release/jobs/deploy-diego-gke-sa/builds/7

Reply:
From the build, https://concourse.suse.dev/teams/main/pipelines/prabal-demo-release/jobs/deploy-diego-gke-sa/builds/7#L5e8e0bc6:13, which comes from https://github.com/SUSE/catapult/blob/540c4c8cfb4992c428bd64ab8c784f1b7a70ac26/modules/common/deps.sh#L23.

So maybe `DOWNLOAD_BINS=false` was incorrectly set in the pipeline, instead of using `DOWNLOAD_CATAPULT_DEPS=false`. 

Nevertheless, I think it's a good idea to add a helm version to the catapult image. But then, it's not a backend dep, therefore my nitpicking comment about the position of the paragraph.